pixiedust/utils/sampleData.py

```diff
@@ -118,6 +119,11 @@ def getType(t):
                 else:
                     return StringType()
 
+        # Force load to pandas
+        if Environment.hasSpark and self.forcePandas:
```

Comment:
rather than duplicate the pandas load/read could we possibly do something like
```
if Environment.hasSpark and not self.forcePandas:
    if Environment.sparkVersion == 1:
        # read/load csv using spark1
    else:
        # read/load csv using spark2
else:
    # read/load csv using pandas
```
otherwise, all looks good

Reply:
Yes, that looks better.  Thanks!